docs/interface.md

```diff
@@ -58,12 +58,14 @@ This mode supports a number of command-line arguments, the details of which can
 
 * `--hf_hub_log_args` : Logs evaluation results to Hugging Face Hub. Accepts a string with the arguments separated by commas. Available arguments:
     * `hub_results_org` - organization name on Hugging Face Hub, e.g., `EleutherAI`. If not provided, the results will be pushed to the owner of the Hugging Face token,
-    * `hub_repo_name` - repository name on Hugging Face Hub, e.g., `lm-eval-results`,
```

Comment:
removing `hub_repo_name` will break backward-compatibility--I wonder whether it would be acceptable to, if `hub_repo_name` is passed, set both `details_` and `results_` repo names to be `hub_repo_name` (and print a logger message to this effect stating that `hub_repo_name` is no longer recommended?)

Reply:
added it back with a warning message !